lib/galaxy/tools/deps/resolvers/__init__.py

```diff
@@ -91,8 +92,12 @@ class NullDependency( Dependency ):
     dependency_type = None
     exact = True
 
+    def __init__(self, version, name):
+        self.version = version
+        self.name = name
+
     def shell_commands( self, requirement ):
         return None
 
 
-INDETERMINATE_DEPENDENCY = NullDependency()
+INDETERMINATE_DEPENDENCY = NullDependency
```

Comment:
Thanks, I found a few more instances where I had to update the code (b59b797).
The reason for the change is that I would like to use the resolver code to display whether a dependency can be resolved, but the NullDependency class is missing the dependencies' name and version.

I can also revert 0b24283, and [return to replicating the resolver logic](https://github.com/galaxyproject/galaxy/pull/2554/commits/0b242833959a5554d60488d6551da78c816020d4#diff-f623c3ca7acff7a0346735bed0aca07aL129) to determine whether a dependency is installed.
I kind of prefer how it is now, but either way is OK for me.